<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstrinsicEdNodeViewClass</class>
 <widget class="QDockWidget" name="InstrinsicEdNodeViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Node View</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QToolButton" name="refreshToolButton">
         <property name="toolTip">
          <string>Refresh Node View</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../IntrinsicEd.qrc">
           <normaloff>:/Icons/icons/arrows/redo.png</normaloff>:/Icons/icons/arrows/redo.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="createRootNodeButton">
         <property name="toolTip">
          <string>Create Root Node</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../IntrinsicEd.qrc">
           <normaloff>:/Icons/icons/cad/circle-2.png</normaloff>:/Icons/icons/cad/circle-2.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="createNodeToolButton">
         <property name="toolTip">
          <string>Create Node</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../IntrinsicEd.qrc">
           <normaloff>:/Icons/icons/cad/cube-1.png</normaloff>:/Icons/icons/cad/cube-1.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nodeCountLabel">
         <property name="text">
          <string>42 Nodes</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="IntrinsicEdNodeViewTreeWidget" name="treeWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IntrinsicEdNodeViewTreeWidget</class>
   <extends>QWidget</extends>
   <header>intrinsicednodeviewtreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../IntrinsicEd.qrc"/>
 </resources>
 <connections/>
</ui>
